dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -182,6 +183,9 @@ internal class RumViewScope(
         )
         writer.write(rumEvent)
         errorCount++
+        if (event.isFatal) {
```

Comment:
I don't see where you mark the sent ErrorEvent as fatal, isn't this supposed to be like that ?

Reply:
`isFatal` is a boolean parameter in the `RumRawEvent.AddError` data class. It's set anytime I create such an event with false/true , eg : 
```
RumRawEvent.AddError(message, source, throwable, false, attributes)
```